src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -1,24 +1,5 @@
 package org.jenkinsci.plugins.mesos;
 
```

Comment:
This diff is hard to read because it's mixing formatting changes with logic changes. But I trust that @kaiwalyajoshi reviewed it properly.